experiments/unified-core/cmd/atc-unified/main.go

```
@@ -0,0 +1,295 @@
+	}
+	remaining := flags.Args()
+	if len(remaining) < 2 {
+		return errors.New("usage: atc-unified api [--base URL] METHOD PATH [JSON]")
+	}
+	var body io.Reader
+	if len(remaining) > 2 {
+		body = strings.NewReader(strings.Join(remaining[2:], " "))
+	} else if remaining[0] != http.MethodGet {
+		body = input
+	}
+	request, err := http.NewRequest(remaining[0], strings.TrimRight(*base, "/")+remaining[1], body)
```

golangci-lint (2.12.2)

[error] net/http.NewRequest must not be called. use net/http.NewRequestWithContext

(noctx)